docs/SetupDatabaseAKS.md

```diff
@@ -0,0 +1,92 @@
+# Set up a database for Scalar DL/Scalar DB deployments in AKS
```

Comment:
```suggestion
# Set up a database for Scalar DB / DL deployments in AKS
```

Reply:
I have not changed it as of now. I am not sure this can be considered as a redundancy.  I will check further and confirm if needed.